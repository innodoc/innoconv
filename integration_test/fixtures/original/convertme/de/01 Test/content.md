---
title: 'A'
---

B  A

A B A
