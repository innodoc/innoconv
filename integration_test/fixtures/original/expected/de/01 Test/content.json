[{"c": [{"c": "B A", "t": "Str"}], "t": "Para"}, {"c": [{"c": "A B A", "t": "Str"}], "t": "Para"}]